pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -370,6 +370,10 @@ func ValidatePriorityLevelConfigurationSpec(spec *flowcontrol.PriorityLevelConfi
 			allErrs = append(allErrs, field.Forbidden(fldPath.Child("limited"), "must be nil if the type is not Limited"))
 		}
 	case flowcontrol.PriorityLevelEnablementLimited:
+		if spec.Exempt != nil {
```

Comment:
Don't we also want to check the values in `NominalConcurrencyShares` and `LendablePercent` in the exempt case, analogously to what ValidateLimitedPriorityLevelConfiguration does?